### DISCUSSION TITLE
-bash: ./gogs: No such file or directory

### DISCUSSION THREAD

**Discussion by aathan** [gogs/gogs#6910] [General]
I just wanted to leave a note here for any others that run into this issue:

If you accidentally download the wrong binary tarball of gogs (.e.g, the 386 vs amd64), then when you try to run `./gogs web` per the install instructions, you'll end up getting the cryptic (bash generated) message:

```
$ ./gogs web
-bash: ./gogs: No such file or directory
```

Checking the gogs executable with `file` will tell you it's the wrong binary:
```
$ file gogs
gogs: ELF 32-bit LSB executable, Intel 80386, version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux.so.2, for GNU/Linux 2.6.32, Go BuildID=TVf0ByqgHIoZsaCpIOO0/9K_XZwos5GCeaF8P_cCm/8bWY9gEYSbBOyKSsXlcY/RJ3V-tvUC7yTuRBNJL87, BuildID[sha1]=a6bc698950e485c8e85b296c55aae77932733733, not stripped
```

**Comment by unknwon** [comment]
Thanks for sharing!

I remember the first time I encounter such error... I was like.. WTF, is bash blind? 😂
